src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -5224,13 +5224,7 @@ int RGWRados::restore_obj_from_cloud(RGWLCCloudTierCtx& tier_ctx,
 
   ceph::real_time restore_time = real_clock::now();
   {
-    char buf[32];
```

Comment:
you missed to copy the changes of rgw_admin.cc here from the commit I gave - https://github.com/soumyakoduri/ceph/commit/37fae2f7058649d5a2f78ec9d5316c05a4d0e2bc

Reply:
Done!